back-end/app.js

```diff
@@ -7,13 +7,20 @@ const { PATHS } = require('./config')
 const productsRoutes = require('./routers/productsRoutes')
 const loansRoutes = require('./routers/loansRoutes')
 
+const auth = require('./middleware/auth')
+
 app.use(cors())
 app.use(express.json())
 app.use(express.static('public'))
 
+app.all('*', auth)
+
 app.use(PATHS.products, productsRoutes)
 app.use(PATHS.loans, loansRoutes)
 
-app.get('*', (req, res) => res.send('Page Not found 404'))
+// eslint-disable-next-line no-unused-vars
+app.use((req, res, next) => {
```

Comment:
@Jeffzholy this should be a route on the other hand, so this could be `app.all('*')`
also you can remove the parameters so you dont need the eslint ignore